### DISCUSSION TITLE
New Laser Master 2 20w will not engrave

### DISCUSSION THREAD

**Discussion by jayronayne** [arkypita/LaserGRBL#1132] [Q&A]
I cannot get my new Ortur Laser Master 2 to engrave using Laser GRBL.  When I hit the "laser" custom button it fires a burn mark into my work piece.  When I attempt to engrave, it seems to run correctly but it doesn't seem to have any power from the laser so it doesn't burn anything at all. 

 What am I missing?  Any help is appreciated.

**Comment by BeatSlayer** [comment]
Does the laser turn on?
Which speed and power did you selected?

**Comment by MaiLes839** [comment]
Hi, I have the same problem. the laser head is going through the motions but doesn't burn the wood. 
There is a laser blue light so I'm assuming the laser is turned on, I tried all sort of power and speed but now I tried the highest one and it still doesn't work
![image](https://user-images.githubusercontent.com/72291216/94983539-57957c80-054c-11eb-91f0-49a76fdecc57.png)

I will appreciate your help, I and struggling with this and can't find the answer anywhere online... thanks!

**Comment by gmmanonymus111** [comment]
I think (by the picture you uploaded) you are setting the laser power at a wrong place. You should set it back to 1x.
You can change laser power during importing an image/vector (I was importing a PNG image):
![image](https://user-images.githubusercontent.com/9658243/94985926-bbbf3d00-055a-11eb-85d6-317cd1d8f4f5.png)
Alternatively, if you want to test it, you can use the following commands

- If you have lasermode enabled (`$32=1` in GRBL->GRBL configuration):
`G1 F1 M3 S*` where * can be any number between 1-255 (or your laser maximum, configured at `$30` (Maximum spindle speed). Set it to 0 to turn your laser off.

- If you have no lasermode (`$32=0`):
`M3 S*` should be enough, the formula is the same as above ^

**Comment by MaiLes839** [comment]
Thank you so much for answering me.
I still can't make it work though, I tried changing the power of the laser where you showed in the image, but still nothing... and 

![image](https://user-images.githubusercontent.com/72291216/94986424-4d34ac00-0567-11eb-8c0c-bdfee42e2f48.png)

I didn't quite understand your last comment...
I only have M3 (no M4), and I have no lasermode ($32=0):

![image](https://user-images.githubusercontent.com/72291216/94986442-7fdea480-0567-11eb-9f05-638365e2eabc.png)

Where do I need to put the formula you wrote on your comment: G1 F1 M3 S* (1000 - this is the number that is in $30).
Do you have any other thoughts on what do I do wrong...?

Thanks!

**Comment by gmmanonymus111** [comment]
You need to type GCode commands here once you are connected to your machine:
![image](https://user-images.githubusercontent.com/9658243/94987624-f67ba200-0567-11eb-88a6-3183f17a1631.png)
According to your pictures, the command should be: `G1F1M3S5000` for 50% power (as max power is set at 10000, not 1000). This should light up your laser

Also, if you intend to engrave with a laser, you should enable laser mode (it'll give you finer images, and when your machine is on pause, GRBL will turn off the laser automatically for safety reasons)

**Comment by MaiLes839** [comment]
OK I tried that - changed to enable laser mode and added the code but still nothing has changed....

![image](https://user-images.githubusercontent.com/72291216/94989714-8547e900-057f-11eb-86d4-69fb54f08359.png)

![image](https://user-images.githubusercontent.com/72291216/94989711-7f520800-057f-11eb-8447-40d746fd0af5.png)

**Comment by gmmanonymus111** [comment]
It looks like the laser is out of focus, the blue dot is too big. Try to rotate the lens/increase or decrease the height of the material you try to engrave, if the laser is a fixed focus one. If it is already focused, I'm afraid the diode is dead.
EDIT: It seems you didn't send the code, just typed it in the text field. You need to send it (aka press enter after typed in), when your machine is idle (so, no running engraving). Like this:
![image](https://user-images.githubusercontent.com/9658243/94990062-78c09200-0579-11eb-9664-175913f03bcd.png)
And because your MAX PWM is set to 10000, the engraving power you set (200-350) is way too low. It means, the laser will use only 2%-3,5% power. Try to increase it to 2000-3500. and run an engraving that way.
![image](https://user-images.githubusercontent.com/9658243/94990164-45cace00-057a-11eb-96e5-4b2f1568e270.png)

**Comment by MaiLes839** [comment]
you are an angel, but still - I tried everything you said and nothing is happening ):

**Comment by gmmanonymus111** [comment]
As a last resort, you should try engraving a grayscale image, like this one:
![grayscale](https://user-images.githubusercontent.com/9658243/94991537-1ae57780-0584-11eb-97f0-f3222c86ade5.png)
The settings should be the following:
![image](https://user-images.githubusercontent.com/9658243/94991678-dc03f180-0584-11eb-9815-f4a3195641f7.png)
![image](https://user-images.githubusercontent.com/9658243/94991760-62203800-0585-11eb-9bab-417bcff9d5a9.png)
This should take ~ 15 mins, but it will go from 0 to 10000, so you can check where (if) it starts to burn the wood.
What model and how old is the engraver by the way?

**Comment by MaiLes839** [comment]
It worked!!!!! can you explain maybe why? so I will understand what I did wrong...?
(it engraved it as a mirror writing though - how come...?)
I just bought the engraver, this is the first time I tried it.... it is a mini CNC 20W 7000 MW STUMP
Thanks so much!!

**Comment by gmmanonymus111** [comment]
Well, I think it needs an effective power range and speed. You need to check where it started burning (it is a 5.5W laser according to the online description, just like mine, so maybe around ~3000 @ 600mm/min). This will be really time consuming, because you have to try a lot of settings. Luckily, LaserGRBL has a material DB (`GRBL`->`Material DB` or click on the book icon when importing an image), which can help you set the correct values.
EDIT: You can use `Ortur LU 20W (5.5W)` as reference
![image](https://user-images.githubusercontent.com/9658243/94993171-56397380-058f-11eb-9496-1c2227c925a9.png)

As for the mirror writing, you might need to change the direction of one axis. (use `$3` at `GRBL`->`GRBL Configuration`, `$2` is just for reference)
![image](https://user-images.githubusercontent.com/9658243/94993374-e75d1a00-0590-11eb-9466-74aa88545b81.png)
More info of the configuration can be found [here](https://github.com/gnea/grbl/wiki/Grbl-v1.1-Configuration)

**Comment by MaiLes839** [comment]
Thanks again for all your explanation, I really appreciate your help!!!!! It was super useful (:

**Comment by gmmanonymus111** [comment]
Glad I could help ^^

**Comment by MaiLes839** [comment]
Alex, do you know what can be the reason for these (unwanted) lines being engraved and how I might be able to avoid them? Thanks
![image](https://user-images.githubusercontent.com/72291216/95704461-910b6d80-0c59-11eb-8dea-8eb0d0d372b4.png)

**Comment by StuartB4** [comment]
Have you tried increasing the White clip slider where you prepare the image.
![image](https://user-images.githubusercontent.com/16980026/95743138-296d1680-0c89-11eb-98c0-62e1a1bc0c06.png)

**Comment by MaiLes839** [comment]
I will give it a try, thanks!
